Content.Client/UserInterface/Systems/ProgressColorSystem.cs

```diff
@@ -0,0 +1,83 @@
+using System.Linq;
+using Content.Shared.CCVar;
+using Robust.Shared.Configuration;
+
+namespace Content.Client.UserInterface.Systems;
+
+/// <summary>
+/// This system handles getting an interpolated color based on the value of a cvar.
+/// </summary>
+public sealed class ProgressColorSystem : EntitySystem
+{
+    [Dependency] private readonly IConfigurationManager _configuration = default!;
+
+    private static bool _colorBlindFriendly;
+
+    private static IReadOnlyList<Color> _plasma = new[]
```

Comment:
This should be stored as a `private static readonly Color[]`, not an `IReadOnlyList<Color>`.